src/index.js

```diff
@@ -51,7 +51,7 @@ Object.assign(Lock.prototype, {
    * Acquire a lock for a specific ID value. Callback returns the following value:
    *
    *   {
-   *     id: Message ID
+   *     id: id of lock
```

Comment:
Can you provide a little more context? Like (if I understand): "arbitrary but should be consistent between servers trying to acquire the lock".